src/plugins/visualizations/public/legacy/build_pipeline.ts

```diff
@@ -202,38 +203,6 @@ export const prepareString = (variable: string, data?: string): string => {
   return `${variable}='${escapeString(data)}' `;
 };
 
-export const prepareValue = (variable: string, data: any, raw: boolean = false) => {
-  if (data === undefined) {
-    return '';
-  }
-  if (raw) {
-    return `${variable}=${data} `;
-  }
-  switch (typeof data) {
-    case 'string':
-      return prepareString(variable, data);
-    case 'object':
-      return prepareJson(variable, data);
-    default:
-      return `${variable}=${data} `;
-  }
-};
-
-export const prepareDimension = (variable: string, data: any) => {
```

Comment:
i think this utility function is still useful to prevent code duplication across all vis types

Reply:
lets just update it to return expression function or undefined